c2rust-analyze/scripts/pointwise_try_build_unmodified.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -euo pipefail
+
+echo
+
+f=$1
+shift 1
+flags=( "$@" )
+echo "f=$f"
+
+name=${f%%.*.rs}
+name=${name##**/}
+echo "name=$name"
+
+func=${f%.rs}
+func=${func##*.}
+echo "func=$func"
+
+filter_errors() {
+    jq 'select(.level == "error") | .message' -r |
+        { grep -v -e '^aborting due to ' -e '^call to unsafe function is unsafe ' || true; }
+}
+
+d="$(dirname "$f")"
+f="$d/${name}_safe_${func}.rs"
+cp "$d/$name.rs" "$f"
```

Comment:
These are the only difference between these two scripts--it would be good to just make these conditional and avoid the duplication.

Reply:
I've updated this to deduplicate the similar scripts.